functions.sh

```diff
@@ -1013,7 +1013,7 @@ validate_vars() {
             graph_error "ERROR: CentOS 32bit doesn't support xfs on partition /"
             return 1
           fi
-          for j in `seq 1 $PART_COUNT`; do
+          for j in $(seq 1 $PART_COUNT); do
```

Comment:
I like the use of seq much more, it contains significantly less non alphanumeric characters.